main.tex

```diff
@@ -521,11 +560,118 @@ \subsection{Over-spending prevention by balance proof}\label{balance}
+ \item[Infrastructure] Several mixing-protocols assume and rely on infrastructure which is not practically available. For example, CoinShuffle assumes a peer-to-peer public bulletin board.
+ \item[Trustlessness] Some protocols have stronger trust assumptions with respect to theft, for example Mixcoin and Blindcoin\footnote{Both protocols have provisions for accountability of the mixes, mitigating theft concerns}, whereas other protocols utilize Bitcoin's scripting capabilities to secure user funds.
+\end{description}
+
+Since Chaumian CoinJoins have been successfully deployed despite some limitations (see \cref{sec:limitations}), it appears that centrally coordinated CoinJoins strike a desirable balance in the Bitcoin privacy design space. It is non-custodial, has low messaging complexity, and results in a single transaction minimizing delays and network fees.
+
+In \cref{table:coinjoin,table:noncoinjoin}, $n$ denotes the number of participants. Overt transactions are apparent and fingerprintable on the blockchain, but still provide privacy within the transaction. Disjoint transactions do not link individual users' inputs and outputs on the blockchain. Covert transactions are not fingerprintable as privacy enhancing transactions.
+
+\begin{table}[h!]
+\begin{minipage}{\textwidth}
+\renewcommand{\thefootnote}{\thempfootnote} % for multiple references to minipage footnotes
+\centering
+\begin{tabular}{lcccccc}
+	&&&& \multicolumn{2}{c}{Privacy against}  \\
+	%\cline{5-6}
+	&Messages & Denominations & Coordination & Server & Participants \\
+
+	Knapsack~\cite{maurer2017anonymous} & - & variable & - & - & -\footnote{Transaction structure guarantees minimal ambiguity in sub-transaction assignments} \\
+	CoinShuffle~\cite{ruffing2014coinshuffle} &$\mathcal{O}(n^2)$ & fixed & decentralized & n.a. & \cmark\footnote{Requires only 2 honest participants.} \\
+	CoinShuffle++~\cite{ruffing2017p2p} & $\mathcal{O}(n)$ & fixed & decentralized & n.a. & \cmark\footnotemark[\value{footnote}] \\
+	CashFusion & $\mathcal{O}(n)$ & variable & centralized & \cmark\footnote{Assuming server does not participate as a verifier.} & \cmark\footnote{Transaction ambiguity guarantees are probablistic, with additional computational difficulty under assumptions.} \\
+	JoinMarket & $\mathcal{O}(n)$ & variable & by taker\footnote{Transactions are initiated and coordinated by users buying mixing offers on a decentralized markeplace.} & n.a.\footnotemark[\value{footnote}] & \cmark \\
+	Chaumian CoinJoin & $\mathcal{O}(n)$ & fixed & centralized & \cmark & \cmark \\
+	This Work & $\mathcal{O}(n)$ & variable & centralized & \cmark & \cmark \\
+\end{tabular}
+\end{minipage}
+\caption{CoinJoin based protocols, no theft risk and requiring a single overt transaction.}
+\label{table:coinjoin}
+\end{table}
```

Comment:
Can you put a screenshot here? I can't review it without looking at it :)

Reply:
![image](https://user-images.githubusercontent.com/14242/86631537-c733b400-bfbd-11ea-9198-368068e2b8e2.png)

Reply:
ACK